app/src/main/java/com/amaze/filemanager/utils/ServiceWatcherUtil.java

```diff
@@ -192,4 +213,24 @@ public void run() {
 
         handler.postDelayed(runnable, 0);
     }
+
+    public interface ServiceWatcherInteractionInterface {
+
+        enum STATE {
```

Comment:
Use final int vars.

Reply:
No, enum s are cool !!

Reply:
They have [a significant overhead](https://android.jlelse.eu/android-performance-avoid-using-enum-on-android-326be0794dc3), not needed here.

Reply:
Done